util/src/main/java/io/grpc/util/OutlierDetectionLoadBalancer.java

```diff
@@ -720,15 +745,15 @@ void maybeUnejectOutliers(Long detectionTimerStartNanos) {
       if (trackerMap.isEmpty()) {
         return 0;
       }
-      int totalAddresses = 0;
-      int ejectedAddresses = 0;
-      for (AddressTracker tracker : trackerMap.values()) {
-        totalAddresses++;
+      int totalEndpoints = 0;
+      int ejectedEndpoints = 0;
+      for (EndpointTracker tracker : trackerMap.values()) {
+        totalEndpoints++;
         if (tracker.subchannelsEjected()) {
-          ejectedAddresses++;
+          ejectedEndpoints++;
         }
       }
```

Comment:
If you like it you could use guava [Iterables](https://guava.dev/releases/21.0/api/docs/com/google/common/collect/Iterables.html)
```
      Iterable<AddressTracker> filteredIterable =
          Iterables.filter(trackerMap.values(), AddressTracker::subchannelsEjected);
      int ejectedEndpoints = Iterables.size(filteredIterable);
```